e2e-tests/gef/cypress/e2e/clone/facility-end-date-enabled/clone-v0-to-v1.spec.js

```diff
@@ -0,0 +1,84 @@
+import relative from '../../relativeURL';
+import { BANK1_MAKER1 } from '../../../../../e2e-fixtures/portal-users.fixture';
+import cloneGEFDeal from '../../pages/clone-deal';
+import mandatoryCriteria from '../../pages/mandatory-criteria';
+import nameApplication from '../../pages/name-application';
+import applicationDetails from '../../pages/application-details';
+import facilities from '../../pages/facilities';
+import { submitButton } from '../../partials';
+import aboutFacilityUnissued from '../../pages/unissued-facilities-about-facility';
+
+/**
+ * NOTE: These tests check the backwards compatibility with in-flight version 0 deals.
+ * A migration may need to be run on production if this test is updated.
```

Comment:
The idea is this protects us against breaking v0 functionality. If this test stops working we should double check this isnt because our version 0 deals need updating. For this test in particular its unlikely to cause issues, but I've added this comment to all tests concerning v0 deals

Reply:
Thanks @MarRobSoftwire I still find the comment a bit confusing though, could we say something like this instead, what do you think?

```js
/**
 * NOTE: These tests check the backwards compatibility with in-flight version 0 deals.
 * If migration is updated for version 0 deals/facilities
 * and therefore this test is updated,
 * we need to check if migration needs to be run in production.
 */
```

Reply:
Sorry @MarRobSoftwire I like the new comment but I'm finding it a bit strange, unless i'm missing a point:

```js
/**
 * NOTE: These tests check the backwards compatibility with in-flight version 0 deals.
 * If this test needs updating because version 0 deals are no longer being supported,
 * we need to check if migration needs to be run in production.
 */
```

This reads to me that it's "If we update the test because version 0 is no longer supported ... we need to check production migration"

But if version 0 is no longer supported, the test would just be removed, no?

I feel it should say "if migration is updated (and therefore this test is updated), we need to check production migration"

Reply:
We're using mongodb & dont have automatic migrations that would be 'updated' - maybe theres a better word to make it clearer?

Thoughts on
``` 
 * If this test needs updating then version 0 deals may no longer be supported and
 * we need to update these deals in production.
```

Reply:
Looks good @MarRobSoftwire - could/should we add the same kind of comment in non-E2E level code?

Reply:
I'll add it to the gef-deal-versioning file